### ISSUE TITLE
Accessibility improvements - Loader spinner

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#5049]
This ticket describes accessibility improvements for the Loader component.

With `aria-live` tags (or `role="status"`) we can communicate the `loading` state to screen readers. For this to work correctly the element with the `role="status"` attribute should be placed in the DOM before the Loader is added. This way the screen reader can notice a change in the DOM and alert the user.

In the current setup, `role="status"` is part of the Loader component. This means that the loading spinner and the a11y tag are added at the same time, which doesn't alert the user. We should create a situation where the DOM always contains an element with `role="status"`, in which we can add the spinner when needed.

- [ ] 4.1.3 (status messages) The loading spinners have issues with aria-live which results into status updates not being announced
